### PR TITLE
fixing 2704 -- Rename LanguageViewLocationExpanderOption to LanguageV…

### PR DESCRIPTION
**Pull request by kulmugdha**
…iewLocationExpanderFormat

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs [hunk 1]
```diff
@@ -25,23 +25,23 @@ namespace Microsoft.AspNet.Mvc.Razor
...
+            : this(LanguageViewLocationExpanderFormat.Suffix)
         {
         }
 
         /// <summary>
         /// Instantiates a new <see cref="DefaultTagHelperActivator"/> instance.
         /// </summary>
-        /// <param name="option">The <see cref="LanguageViewLocationExpanderOption"/>.</param>
-        public LanguageViewLocationExpander(LanguageViewLocationExpanderOption option)
+        /// <param name="option">The <see cref="LanguageViewLocationExpanderFormat"/>.</param>
```
Comment (kirthik) [inline src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs]: change param name to `format`

File: src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpanderOption.cs [hunk 2]
```diff
@@ -6,10 +6,10 @@ namespace Microsoft.AspNet.Mvc.Razor
     /// <summary>
     /// Specifies the localized view format for <see cref="LanguageViewLocationExpander"/>.
     /// </summary>
-    public enum LanguageViewLocationExpanderOption
```
Comment (kirthik) [inline src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpanderOption.cs]: change the file name to `LanguageViewLocationExpanderFormat`

**Comment by kirthik** [comment]
:shipit:  after addressing minor comments.

**Comment by kulmugdha** [comment]
Thanks @kirthik ! Checked in 8cb2aae4f5ce7899c123e41bdc53b7ab0e9a3008.

**Pull request closed without merging**